test/api_bridge_test.rb

```diff
@@ -27,7 +29,7 @@ def before_setup
     facets = ["creator.name", "date.year", "subcategory"]
     assert_equal facets, @api.app_facets
     assert_equal(
-      {"num"=>5, "sort"=>"title|asc", "f"=>["collection|letters"], "facet"=>["creator.name", "date.year", "subcategory"]},
+      {"num"=>5, "sort"=>"title|asc", "f"=> @f_app, "facet"=>["creator.name", "date.year", "subcategory"]},
```

Comment:
Do you want to re-use `@facet_app` here with `"facet" => @facet_app` similar to `"f" => @f_app`?